spec/features/clearance/visitor_signs_up_spec.rb

```diff
@@ -11,19 +11,19 @@
   end
 
   scenario "with valid email and password" do
-    sign_up_with "valid@example.com", "password"
+    sign_up_with "valid@example.com", "password", "Michael", "I am Michael", "01/01/1970", "yoga, crack", "Earth"
```

Comment:
Doesn't feel like all of these attributes are important for this test. Can you make them optional?

Reply:
everything except 'email' and 'password' is now optional